code/modules/admin/holder2.dm

```diff
@@ -26,6 +26,8 @@ GLOBAL_PROTECT(href_token)
 	/// Our index into GLOB.antagonist_teams, so that admins can have pretty tabs in the Check Teams menu.
 	var/team_switch_tab_index = 1
 
+	var/datum/bloom_edit/debug_bloom
```

Comment:
please autodoc this, unless it's purely for debugging purposes, in which case you probably want to remove this?